include/seqan3/search/fm_index/fm_index.hpp

```diff
@@ -178,12 +178,26 @@ class fm_index
     /*!\name Constructors, destructor and assignment
      * \{
      */
-    fm_index() = default;                             //!< Default constructor.
-    fm_index(fm_index const &) = default;             //!< Copy constructor.
-    fm_index & operator=(fm_index const &) = default; //!< Copy assignment.
-    fm_index(fm_index &&) = default;                  //!< Move constructor.
-    fm_index & operator=(fm_index &&) = default;      //!< Move assignment.
-    ~fm_index() = default;                            //!< Destructor.
+    fm_index() = default;              //!< Defaulted.
+    fm_index(fm_index const & rhs)     //!< When copy constructing, also set the pointers of rank and select support.
```

Comment:
Do users of the fm_index (need to) know what rank and select support are?

Reply:
`When copy constructing, also update internal data structures.` ?